sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -152,47 +152,20 @@ Git Hooks
 ~~~~~~~~~
 
 In order to help prevent some accidental errors, here are some git hooks
-that may be useful. The scripts below should be placed in the ``.git/hooks``
-subdirectory in the top level of the *source checkout* directory and be
-marked executable with e.g. ``chmod +x pre-commit``. For more information
-on git hooks, see `this tutorial`_.
+that may be useful:
 
-``pre-commit``
+- Run codebase tests
+  This git hook runs all the codebase tests before allowing a push to the remote
```

Comment:
```suggestion
- Run codebase tests

  This git hook runs all the codebase tests before allowing a push to the remote
```
I believe a blank line is necessary to have the bullet text render on a line of its own, with a para below.